### PR TITLE
Diala Alghamdi

### PR DESCRIPTION
**Pull request by Dialaj**
* on a scale from 1 to 5, how comfortable were you with this assignment? 5
 * How long did it take? 1 hour
 * What was a challenge you had with this assignment? ---
 * Is there anything that you'd like some further information on? No thank you 
 * Do you have any suggestions to improve this assignment? No thank you

### CODE REVIEW

**Comment by i-am-ebere** [comment]
Thats awesome you solved it but who is this?

**Comment by Dialaj** [comment]
> Thats awesome you solved it but who is this?

Hello It's me

File: Homework.rb [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+    },
+    {
+        first_name: "Doaa",
+        last_name: "Alalawi",
+    }
+  ]
+  upper_case_full_names = []
+
+  for s in students do
+    upper_case_full_names.push("#{s[:first_name].upcase} #{name[:last_name].upcase}")end
```
Comment (yasser9947) [inline Homework.rb:21]:
```
    upper_case_full_names.push("#{s[:first_name].upcase} #{name[:last_name].upcase}")end
```
I think name is  undefind  so you will replace the name to s 

```
    upper_case_full_names.push("#{s[:first_name].upcase} #{s[:last_name].upcase}")end

```

**Review by yasser9947 — COMMENTED** [review]
Well done Diala
Very good work, great development and very noticeable for your level of programming ماشاء الله,
Now we can say that you picked some of the fruits of your diligence, always forward Diala 💯 👍 
 
Greetings H3RO

**Pull request closed without merging**